src/main/java/net/pms/dlna/DVDISOFile.java

```diff
@@ -90,7 +109,7 @@ protected void resolveOnce() {
 			"null",
 			"-dvd-device",
 			ProcessUtil.getShortFileNameIfWideChars(file.getAbsolutePath()),
-			"dvd://1"
+			"dvd://"
```

Comment:
After all, perhaps it's what would prefer the users ?
So that's should be fine.